netbout-web/src/test/java/com/netbout/rest/MarkdownTest.java

```diff
@@ -39,6 +39,26 @@
 public final class MarkdownTest {
 
     /**
+     * Markdown can handle whitespace after links.
+     * @throws Exception If there is some problem inside
+     */
+    @Test
+    public void handleWithespaceAfterLinks() throws Exception {
```

Comment:
@caarlos0 I'll use `handlesWhitespaceAfterLinks` because whitespace is uncountable.